src/clj/vanilla/db/core.clj

```diff
@@ -3,7 +3,9 @@
     [hugsql.core :as hugsql]
     [next.jdbc :as jdbc]
     [next.jdbc.sql :as sql]
-    [com.stuartsierra.component :as component]))
+    [com.stuartsierra.component :as component]
+    [clojure.java.io :as jio]
+    [clojure.tools.logging :as log]))
```

Comment:
figured adding logging would be useful, especially given the upcoming "service-log" PR